srcpkgs/python3-Pillow/template

```diff
@@ -14,7 +14,11 @@ license="custom:PIL"
 homepage="https://python-pillow.org"
 changelog="https://raw.githubusercontent.com/python-pillow/Pillow/master/CHANGES.rst"
 distfiles="${PYPI_SITE}/P/Pillow/Pillow-${version}.tar.gz"
-checksum=59304c67d12394815331eda95ec892bf54ad95e0aa7bc1ccd8e0a4a5a25d4bf3
+checksum=11c5c6e9b02c9dac08af04f093eb5a2f84857df70a7d4a6a6ad461aca803fb9e
+
+do_check() {
+	PYTHONPATH=$(cd build/lib.linux-* && pwd) ./selftest.py
```

Comment:
The makefile inlcuded in the tarball uses `pytest` to run the `test` target. Add `python3-pytest` to `checkdepends` and replace `./selftest.py` with `python3 -m pytest`. This will run more thorough tests. You can keep the subshell trick to set `PYTHONPATH`, which will still be required.

Reply:
thanks, some tests fail, but they should do not seem to be mandatory

Reply:
The (two) tests that fail complain about current implementations not being precise enough. This reads more like a limitation of an implementation issue in Pillow or the Python stdlib rather than some packaging error. I'm not worried about it.